gtsfm/configs/sift_front_end.yaml

```diff
@@ -3,7 +3,7 @@ SceneOptimizer:
   save_gtsfm_data: True
   save_two_view_correspondences_viz: False
   save_3d_viz: True
-  pose_angular_error_thresh: 10 # degrees
+  pose_angular_error_thresh: 5 # degrees
```

Comment:
this name could be made more clear, now that we have so many thresholds

Reply:
Hmm I'm having trouble thinking of something clearer. DId you have something specific in mind?

Reply:
I meant conveying that pose angular error is max of rotation and direction angular error. Or maybe its too verbose?

Reply:
yeah i fear it would be too verbose

Reply:
Do you think we should add the keyword `metric` in the name. We might have thresholds based on cycle consistency or consistency between global-local pose values. Maybe something like `pose_angular_error_metric_thresh`?

Reply:
Thanks for the review Ayush.

I think `pose_angular_error_metric_thresh` is a bit too long. I think we could later add `max_cycle_rot_deviation_deg` and `max_global_relative_deviation_deg`. But I prefer the current names.